src/routes/event.ts

```diff
@@ -409,33 +411,33 @@ router.put(
                     : undefined,
             };
             let diffText =
-                "<p>This event was just updated with new information.</p><ul>";
+                "<p>" + i18next.t("routes.event.difftext") + "</p><ul>";
             let displayDate;
             if (event.name !== updatedEvent.name) {
-                diffText += `<li>the event name changed to ${updatedEvent.name}</li>`;
+                diffText += `<li>` + i18next.t("routes.event.namechanged") + updatedEvent.name + `</li>`;
```

Comment:
How would you feel about passing these strings (`updatedEvent.name` and the others) to `i18next.t()` as a second param (`i18next.t("routes.event.namechanged", { name: updatedEvent.name })`), for symmetry with the other places it's used?